lib/storage/src/content_manager/consensus/consensus_callbacks.rs

```diff
@@ -0,0 +1,18 @@
+
+
+
+pub fn deactivate_replica(
```

Comment:
It looks like a duplicate of the addition in `content_manager/mod.rs`.

Reply:
yes